tests/TestCase/Shell/RoutesShellTest.php

```diff
@@ -58,7 +46,22 @@ public function tearDown()
     {
         parent::tearDown();
         Router::reload();
-        unset($this->io, $this->shell);
+    }
+
+    /**
+     * Check that a row of cells exists in the output.
+     *
+     * @param array $row The row of cells to check
+     * @return void
+     */
+    protected function assertOutputContainsRow(array $row)
+    {
+        $row = array_map(function ($cell) {
+            return preg_quote($cell, '/');
+        }, $row);
+        $cells = implode('\s+\|\s+', $row);
+        $pattern = '/' . $cells . '/';
+        $this->assertOutputRegexp($pattern);
```

Comment:
I think this is a pretty useful assertion - do you think it should be moved to the test case? Also, should be named `assertOutputContainsRows` (plural) as it checks for multiple rows.

Reply:
I can move it up.

Reply:
Won't a plural method name imply it can check for multiple rows? Right now I think it will only find one row.